### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by resdenia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# HELLOY OUR FIRST PROJECT !!
```
Comment (Sashikbear) [inline README.md]: awesome!

**Review by Sashikbear — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (resdenia) [reply README.md]: Thank you!!

Comment (Omriruvio) [reply README.md]: Good, lets go! we have a rough deadline!

**Review by meidankaye — APPROVED** [review]
I figured it out! Great readme

**Pull request merged**